includes/core/admin/metaboxes/settings.php

```diff
@@ -178,22 +205,17 @@ public static function save( $post_id, $post ) {
 			return;
 		}
 
-		// See what type of currency we are dealing with so we know how to save the values
-		$is_zero_decimal = simpay_is_zero_decimal();
-
-
 		/** Payment Options */
 
 		// Amount Type
 		$amount_type = isset( $_POST['_amount_type'] ) ? esc_attr( $_POST['_amount_type'] ) : 'one_time_set';
 		update_post_meta( $post_id, '_amount_type', $amount_type );
 
+		// TODO Rewrite. Hard to read.
+
 		// Amount
-		if ( $is_zero_decimal ) {
-			$amount = isset( $_POST['_amount'] ) ? sanitize_text_field( $_POST['_amount'] ) : ( false !== get_post_meta( $post_id, '_amount', true ) ? get_post_meta( $post_id, '_amount', true ) : '100' );
-		} else {
-			$amount = isset( $_POST['_amount'] ) ? sanitize_text_field( $_POST['_amount'] ) : ( false !== get_post_meta( $post_id, '_amount', true ) ? get_post_meta( $post_id, '_amount', true ) : '1' );
-		}
+		$amount = isset( $_POST['_amount'] ) ? sanitize_text_field( $_POST['_amount'] ) : ( false !== get_post_meta( $post_id, '_amount', true ) ? get_post_meta( $post_id, '_amount', true ) : simpay_global_minimum_amount() );
```

Comment:
This should be double checked.

Reply:
Looks correct. Part of amount conversion refactoring done in early 2018 in Pro.
https://github.com/wpsimplepay/WP-Simple-Pay-Pro-3/commit/fc7474f73217f19868c735b2b78d67e8568a98e2#diff-e560c0ff2a130a6267e5401b99a13c37